react/tic-tac-toe/src/app/screens/Game/components/MoveButton/index.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function MoveButton({ desc, move, onClick }) {
+  function handleClick() {
+    onClick(move);
+  }
```

Comment:
Use arrow functions for all the methods, with the exception of the arrow functions

Reply:
i think this was because of the linter